hudi-utilities/src/test/resources/log4j2-surefire.properties

```diff
@@ -0,0 +1,51 @@
+###
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+status = warn
+name = PropertiesConfig
+
+# CONSOLE is set to be a ConsoleAppender.
+appender.console.type = Console
+appender.console.name = consoleLogger
+# CONSOLE uses PatternLayout.
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %-4r [%t] %-5p %c %x - %m%n
```

Comment:
Hi @hddong , thanks for your contribution.
Can you explain why you configured two formats?  If there is no special reason, keep them in the same format might be better.
The rest  LGTM cc@yanghua